src/core/eme/types.ts

```diff
@@ -30,6 +30,11 @@ export interface IEMEWarningEvent {
   value : ICustomError|Error;
 }
 
+export type IEMEInitEvent = {
+  type: "eme-init";
+  value: IMediaKeysInfos;
+} | { type: "eme-disabled" };
```

Comment:
From what I get, ``"eme-disabled"`` is not at all an event from ``EME``. It's entirely from create by `Init`